### ISSUE TITLE
Jupyterlab Math service does not render plotly

### ISSUE THREAD

**Issue by pcrespov** [ITISFoundation/osparc-simcore#1584]

**Comment by pcrespov** [comment]
Extension is  probably missing. SEE https://stackoverflow.com/a/54066746

```cmd
jupyter labextension list
JupyterLab v2.0.1
Known labextensions:
   app dir: /opt/conda/share/jupyter/lab
        @jupyter-widgets/jupyterlab-manager v2.0.0  enabled  OK
        @jupyterlab/git v0.20.0  enabled  OK
        jupyter-matplotlib v0.7.2  enabled  OK
        nbdime-jupyterlab v2.0.0  enabled  OK
```

**Comment by pcrespov** [comment]
@drniiken can you please paste here the example you tried and was not rendering?
THX

**Comment by pcrespov** [comment]
https://git.speag.com/oSparc/sparc-internal/-/merge_requests/87

**Comment by pcrespov** [comment]
@drniiken the fix is part of released 1.2.2 of jupyterlab-octave+python math service
